addons/calendar/tests/test_event_notifications.py

```diff
@@ -124,8 +124,8 @@ def test_bus_notif(self):
                 "alarm_id": alarm.id,
                 "event_id": self.event.id,
                 "title": "Doom's day",
-                "message": self.event.display_time,
-                "timer": 20*60,
+                "message": self.event.display_time + " ",
+                "timer": 1200,
```

Comment:
I think it was 20 * 60 on purpose to emphasize the fact that it's 20 minutes.
So if you don't need to change those lines, I would leave them as before.

Reply:
done